lib/spack/docs/tables/system_prerequisites.csv

```diff
@@ -2,6 +2,7 @@ Name, Supported Versions, Notes, Requirement Reason
 Python, 2.7/3.5-3.9, , Interpreter for Spack
 C/C++ Compilers, , , Building software
 make, , , Build software
+ninja, , , Build software
```

Comment:
This isn't a system prereq, Spack installs it itself

Reply:
I wasn't sure if it was required for bootstrapping, good to know.